client/src/components/runs/logs/Log.js

```diff
@@ -1368,17 +1368,22 @@ class Logs extends localization.LocalizedReactComponent {
       let kubeServices;
       if (this.endpointAvailable) {
         const urls = this.props.multiZone.getDefaultRunServiceUrls(this.props.run.value.serviceUrl);
-        // todo: use regionedUrls
-        // const regionedUrls = this.props.multiZone.parseRunServiceUrlConfiguration(this.props.run.value.serviceUrl);
-        // console.log(regionedUrls, regionedUrls.map(url => this.props.multiZone.getDefaultURLRegion(url.url)));
+        const regionedUrls = this.props.multiZone.parseRunServiceUrlConfiguration(this.props.run.value.serviceUrl);
         endpoints = (
           <tr style={{fontSize: '11pt'}}>
-            <th style={{verticalAlign: 'top'}}>{urls.length > 1 ? 'Endpoints: ': 'Endpoint: '}</th>
+            <th style={{verticalAlign: 'middle'}}>{urls.length > 1 ? 'Endpoints: ' : 'Endpoint: '}</th>
             <td>
               <ul>
                 {
-                  urls.map((url, index) =>
-                    <li key={index}><a href={url.url} target="_blank">{url.name || url.url}</a></li>
+                  regionedUrls.map((url, index) =>
+                    <li key={index}>
+                      <RegionsDropdown
+                        title={url.name || url.url}
+                        regions={Object.assign({}, ...regionedUrls.map(({url}) => url))}
```

Comment:
`Object.assign(...)` will return wrong `region`:`url` configuration - it will join **all** configurations for all endpoints instead of the configuration for particular one

Reply:
@rodichenko  Done!